PORTAL/jobs/_utils.py

```diff
@@ -1418,16 +1418,16 @@ def render(self):
 # git utils
 
 GITHUB_REMOTE_URL = re.compile(r'''
-    ^(
-        (
+    ^(?:
```

Comment:
This regex was used in other places only to validate a match, not to actually parse it into its parts.  Adding the "non-matching groups" makes this always return 4 results, which is what the existing code assumes.